.github/codeql/codeql-config.yml

```diff
@@ -2,4 +2,15 @@
 name: "CodeQL config"
 
 queries:
- - uses: security-extended  # security-and-quality (very verbose)
+ - uses: security-and-quality # (very verbose)
```

Comment:
Is this a superset of the previous setting

Reply:
Yes.